templates/build-pod/template.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
@InfoSec812 ok, good to know, so in that case I'd recommend changing line 7 below as it reads `Maven Build Pod` (which is what threw me off...)